doc/manual/src/language/syntax.md

```diff
@@ -503,6 +503,24 @@ three kinds of patterns:
     > [ 23 {} ]
     > ```
 
+    The full function argument bound by the `@`-pattern is also brought into scope over the set pattern on the other side of the `@`, not just over the body of the function.
```

Comment:
```suggestion
    The full function argument bound by the `@`-pattern is also brought into scope over the set pattern on the other side of the `@`.
```

Maybe the additional contrast is not even necessary, it makes for a very long sentence.

Reply:
Full function argument isn't accurate. It could be read as (original + defaults).
s/full/original would already be better, but how about this?

```suggestion
    All bindings introduced by the function are available in the `?`-defaults, including the original argument from an `@`-pattern:
```

Reply:
```suggestion
    All bindings introduced by the function are in scope in the entire function expression; not just in the body. For example, the binding of an `@` pattern is available in the `?`-defaults:
```

Reply:
Ah we can in fact break this up: we can document the general rule above with something like `{ a ? b, b ? a }: ...`, and then document that the same thing also applies to `@` bindings here.

Reply:
The simple case with just `?` is already documented above

Reply:
wait never mind, it's not

Reply:
I found @roberth sentence easier to understand.